docs/source/matchers_tutorial.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "=====================\n",
+    "Working with Matchers\n",
+    "=====================\n",
+    "Matchers provide a flexible way of comparing LibCST nodes in order to build more complex transforms. See :doc:`Matchers <matchers>` for the complete documentation.\n",
+    "\n",
+    "Basic Matcher Usage\n",
+    "===================\n",
+    "Lets say you are visiting a LibCST ``Call`` node and you want to know if all arguments provided are the literal ``True`` or ``False``. You look at the documentation and see that ``Call.args`` is a sequence of ``Arg``, and each ``Arg.value`` is a ``BaseExpression``. In order to verify that each argument is either ``True`` or ``False`` you would have to first loop over ``node.args``, and then check ``isinstance(arg.value, cst.Name)`` for each ``arg`` in the loop before finally checking ``arg.value.value in (\"True\", \"False\")``. \n",
```

Comment:
Can you link Call to :class:`~libcst.Call`? That's easier for user who don't familiar with CSTNode.

Reply:
Same comment for Arg and BaseExpression.

Reply:
Will change.